docs/marketplace/dev/setup/marketplace-supported-browsers.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Marketplace supported browsers
+description: This documents lists browsers which the Spryker Marketplace supports.
+last_updated: Jun 8, 2022
+template: howto-guide-template
+---
+
+This document lists browsers which the Spryker Marketplace supports.
+
+### Supported browsers
+
```

Comment:
Consider removing lines 7-10. They do not provide value, the page heading speaks for itself